jobs/elasticsearch/spec

```diff
@@ -41,6 +41,12 @@ properties:
   elasticsearch.health.interval:
     description: Post-start interval for node to join cluster
     default: 15
+  elasticsearch.health.connect_timeout:
+    description: Post-start timeout for node to become available
```

Comment:
The descriptions should say what these units are.  It's impossible to know without reading the source of the scripts that use them